release-notes/opensearch.release-notes-2.1.0.md

```diff
@@ -0,0 +1,140 @@
+## 2022-07-07 Version 2.1.0 Release Notes
+
+
+### Breaking Changes in 2.1.0
```

Comment:
None of these changes appear to be breaking to me, this looks like copy/paste error, we should update and remove this.